### PR TITLE
Fix error-reference for error 5035

### PR DESCRIPTION
**Pull request by jez**
### Test plan

See included automated tests.

### CODE REVIEW

File: website/docs/error-reference.md [hunk 1]
```diff
@@ -1993,18 +1993,27 @@ intent to create a new type alias.)
...
-
-If the parent method definitely exists at runtime, it might be hidden in a
-[`# typed: ignore`](static#file-level-granularity-strictness-levels) file.
-Sorbet will not see it and this error will be raised. In that case you will need
-to either raise the `typed` sigil of that file above `ignore`, or generate an
-[RBI file](rbi) that contains signatures for the classes and methods that file
-defines.
+There are two cases when Sorbet might produce this error code:
+
+1.  A method was marked `override`, but sorbet was unable to find a method in
```
Comment (ilyailya) [inline website/docs/error-reference.md:1998]:
```suggestion
1.  A method was marked `override`, but Sorbet was unable to find a method in
```

File: website/docs/error-reference.md [hunk 2]
```diff
@@ -1993,18 +1993,27 @@ intent to create a new type alias.)
...
+    overridden exists in the ancestors of the class defining the `override`
+    method, or remove `override` from the signature that's raising the error.
+
+    If the parent method definitely exists at runtime, it might be hidden in a
+    [`# typed: ignore`](static#file-level-granularity-strictness-levels) file,
+    or defined via metaprogramming. To fix, either raise the `typed` sigil of
+    that file above `ignore`, or generate an [RBI file](rbi) that contains
+    signatures for the classes and methods that file defines.
+
+1.  Sorbet found the method that this method overrides, but the override method
```
Comment (ilyailya) [inline website/docs/error-reference.md:2009]:
```suggestion
2.  Sorbet found the method that this method overrides, but the override method
```

**Review by ilyailya — APPROVED** [review]
LGTM!

**Comment by limck620** [comment]
> 
> 
> 
> ### Motivation
> 
> 
> 
> ### Test plan
> 
> 
> See included automated tests.

**Pull request merged**